tools/resourcedocsgen/pkg/docs/gen_method.go

```diff
@@ -104,7 +105,7 @@ func (mod *modContext) genMethod(r *schema.Resource, m *schema.Method) methodDoc
 
 	docInfo := dctx.decomposeDocstring(f.Comment, resourceSnippetLanguages)
 	args := methodDocArgs{
-		Title: title(m.Name, ""),
+		Title: strings.Title(m.Name), //nolint:staticcheck // Maintain usage of strings.Title for backwards compatibility.
```

Comment:
Does the deleted `title` method just call `strings.Title`? Or is there a possible change of behaviour here?